backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -168,36 +168,49 @@ def get_filters(self, filters):
         filters_map = {
             "build": [],
             "boot": [],
-            "test": []
+            "test": [],
+            "global": [],
         }
 
         for key, value in filters.items():
-            _, filter_item = key.split("_")
+            key = key.replace("filter_", "")
 
-            filter_split = filter_item.split(".")
-            if len(filter_split) == 2:
-                table, field = filter_item.split(".")
+            table = "global"
+            if "." in key:
+                table, field = key.split(".")
             else:
-                table = "build"
-                field = filter_item
+                field = key
 
-            if table == "build":
+            if field == "valid":
+                table = "build"
                 value = [x == "Success" for x in value]
             filters_map[table].append({"field": field, "value": value})
 
         return filters_map
 
-    def pass_in_filters(self, data, filters):
+    def pass_build_filter(self, data, filters):
         for currentFilter in filters:
             field = currentFilter.get("field")
             value = currentFilter.get("value")
+            include_null = "null" in value or "Unknown" in value
```

Comment:
you could extract these strings to a const at the top and do something like `include_null = value in null_strings` with `NULL_STRINGS = ["null", "Unknown"]`